test/index.js

```diff
@@ -571,7 +571,7 @@ describe('Definitions', () => {
 
                 expect(err).to.exist();
                 expect(err.message).to.equal('Invalid cookie value');
-                expect(err.data).to.deep.equal([
+                expect(err.data).to.deep.include([
```

Comment:
I don't have a better idea how to test this in an elegant way. The value of `reason` is different on node v4 and v6:

- 4: "Unexpected token ]"
- 6: "Unexpected token ] in JSON at position 0"

Any ideas?